cmd/synthetic-monitoring-agent/main.go

```diff
@@ -209,6 +213,19 @@ func run(args []string, stdout io.Writer) error {
 		zl.Info().Msg("disabling k6 checks")
 	}
 
+	var usageReporter *usage.UsageReporter
+
+	if !config.DisableUsageReports {
+		zl.Info().Str("usage endpoint", usage.DefaultUsageStatsEndpoint).Msg("Starting usage stats reporting")
+		endpoint := usage.DefaultUsageStatsEndpoint
+		if config.DevMode {
+			// TODO: Make this point to staging
+			endpoint = "http://localhost:4200"
```

Comment:
Does this mean there's a staging usage stats endpoint? I'm not sure whether we should make a distinction here, just to understand it better, what was your reasoning for this?

Reply:
I think an argument could be made to use staging for local development. Though I think for local development we should automatically disable reporting so we're not storing garbage usage reports.

Reply:
My incination is to remove the ability to set the endpoint until we have a good reason to do so. The usage reporter is either on and sending to stats.grafana.com, or it's off.